helpers/gestures.js

```diff
@@ -0,0 +1,109 @@
+const maxTime = 2000;
```

Comment:
This is the same impl as before, with `var` replaced, and using arrow functions where appropriate.